File: PRAK601-2310817310008-MUHAMMADRIZKIRAMADHAN.py
def matriks(baris, kolom, data):
    mat = [[0 for _ in range(kolom)] for _ in range(baris)]
    indeks = 0

    for i in range(baris):
        for j in range(kolom):
            mat[i][j] = data[indeks]
            indeks += 1
    for i in range(baris):
        for j in range(kolom):
            print(mat[i][j], end=" ")
        print()

baris, kolom = map(int, input().split())
jumlah_data = baris * kolom
data = list(map(int, input().split()))

matriks(baris, kolom, data)

File: PRAK601-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>

void matriks(int baris, int kolom, int data []){
  int mat [baris][kolom];
  int indeks = 0;

  for (int i = 0; i < baris; i++){
    for (int j = 0; j < kolom; j++){
      mat[i][j] = data[indeks];
      indeks++;
    }
  }
  for (int i =  0; i < baris; i++){
    for (int j = 0; j < kolom; j++){
      printf("%d ", mat[i][j]);
    }
    printf("\n");
  }
}
  int main(){
    int baris, kolom;
    printf(" ");
    scanf("%d %d", &baris, &kolom);

    int jumlahdata = baris * kolom;
    int data [jumlahdata];

    printf(" ");
    for(int i = 0; i < jumlahdata; i++){
      scanf("%d", &data[i]);
    }
    matriks(baris, kolom, data);

    return 0;
  }